app/models/katello/repository.rb

```diff
@@ -1050,6 +1054,12 @@ def remove_docker_content(manifests)
       DockerMetaTag.cleanup_tags
     end
 
+    def allow_updates?
+      # allow the update if this repo is not in the default view
+      return true unless in_default_view?
+      root.allow_updates?(ALLOWED_UPDATE_FIELDS)
```

Comment:
Even though it isn't strictly needed, it may be more clear to be explicit about which constant we're using, since they're both called `ALLOWED_UPDATE_FIELDS`.

```suggestion
      root.allow_updates?(::Katello::Repository::ALLOWED_UPDATE_FIELDS)
```

Reply:
Nice, +1 to that.